src/test/kotlin/no/nav/syfo/testutil/TestDB.kt

```diff
@@ -60,6 +65,33 @@ fun Connection.getPersonOppgaveList(fodselnummer: PersonIdent): List<PPersonOppg
     }
 }
 
+fun Connection.getAllPersonoppgaver(): List<PPersonOppgave> {
+    val query = "SELECT * FROM person_oppgave"
+    return use { connection ->
+        connection.prepareStatement(query).use {
+            it.executeQuery().toList { toPPersonOppgave() }
+        }
+    }
+}
+
+fun Connection.getAllMotesvar(): List<PDialogmotesvar> {
+    val query = "SELECT * FROM motesvar"
+    return use { connection ->
+        connection.prepareStatement(query).use {
+            it.executeQuery().toList { toPDialogmotesvar() }
+        }
+    }
+}
+
+fun Connection.getAllDialogmoteStatusendring(): List<PDialogmoteStatusendring> {
+    val query = "SELECT * FROM dialogmote_statusendring"
+    return use { connection ->
+        connection.prepareStatement(query).use {
+            it.executeQuery().toList { toPDialogmoteStatusendring() }
+        }
+    }
+}
```

Comment:
Fantes ingen av disse SELECTene fra før?

Reply:
Bra spørsmål! Nei, ikke på å hente ut alt fra tabellen. Tanken er også å prøve å ikke bruke for mye prodkode i set up og assert delen av testen.